### ISSUE TITLE
BUG: Multiple Callouts/Blockquotes Merged if They End in an Empty Indicator an Empty Line and Then Another Callout or Blockquote Follows Them

### ISSUE THREAD

**Issue by matt-student** [platers/obsidian-linter#684]
Multiple callouts become condensed into one when the 'empty line around blockquotes' setting is on. The expected result after running the linter would be for the '>' not to be inserted into the document. This new bug essentially removes blank lines between multiple callouts and was introduced with the latest update (V1.11.0)

Before Linter
```
## HEADING 2
> [!TLDR] Titel
> blah blah blah

> [!FAQ] Title
> blah blah blah

> [!NOTES] Title
> blah blah blah
```

After Linter
```
## HEADING 2
> [!TLDR] Titel
> blah blah blah
> 
> [!FAQ] Title
> blah blah blah
>
> [!NOTES] Title
> blah blah blah
```

**Comment by pjkaufman** [comment]
I remember fixing this prior, but maybe I did not catch a case for (#672 ). Could you provide your `data.json` so I can see about reproducing the issue?

**Comment by matt-student** [comment]
My bad, I didn't see that issue and there was an update sitting there that I had not installed.

**Comment by pjkaufman** [comment]
No worries

**Comment by matt-student** [comment]
Hi, this issue appears to be back. It appears to be associated with the empty line around the blockquote setting. I have just disabled this for now.

**Comment by pjkaufman** [comment]
I will see if I can reproduce this sometime later, but I just tried with the example on this ticket and could not reproduce. Is it happening for the example provided above?

**Comment by matt-student** [comment]
Forgive me for the weird way of taking notes, it might be the reason I have encountered this issue. I sometimes use callouts to draw extra attention to something when I am studying. I have provided a new example of the scenario which cause the issue. Please let me know if this helps. I can provide my data.json file if you would like that.

```
> [!INFO] Title
> Content

> [!IMPORTANT] Point to emphasise
> 

> [!EXAMPLE] Title
> Content

> [!TLDR] Title
> Content

> [!FAQ] Title
> 

> [!NOTES] Title
> Content
```

So the issue is resolved for the most part... But in this example, the 'point the emphasise' callout will be broken and get bunched together like so:

```
> [!INFO] Title
> Content

> [!IMPORTANT] Point to emphasise
>
> [!EXAMPLE] Title
>
> Content

> [!TLDR] Title
> Content

> [!FAQ] Title
>
> [!NOTES] Title
>
> Content
```

**Comment by pjkaufman** [comment]
I believe that I have a fix for this. I will update the title of this bug and create a PR to fix the issue.

**Comment by pjkaufman** [comment]
Looks like the change I made creates some problems for some other rules, so I need to double check what is going on and  then once things are working get that in. It will likely need to wait until Friday before I can come back to this.

**Comment by pjkaufman** [comment]
I believe I have the fix for this. It does not seem too bad to implement.
